src/ensembl/src/content/app/species/sample-data.ts

```diff
@@ -14,30 +14,615 @@
-    total_sequence_length: 4537931177
+import { SpeciesStatsSection } from 'src/content/app/species/state/general/speciesGeneralHelper';
+
+type RawSpeciesStats = {
+  [genomeId: string]: {
+    [key in SpeciesStatsSection]: {
+      [key: string]: string;
+    };
+  };
+};
+
+export const sampleData: RawSpeciesStats = {
+  homo_sapiens_GCA_000001405_28: {
+    assembly_stats: {
+      scientific_name: 'Homo sapiens',
+      sex: 'NA',
+      breed_cultivar_strain: 'NA',
+      taxonomy_id: '9606',
+      assembly_name: 'GRCh38.p13',
+      assembly_accession: 'GCA_000001405.28',
+      assembly_date: '2019-02-28',
+      contig_n50: '56413054',
+      total_genome_length: '3272116950',
+      total_coding_sequence_length: '34317174',
+      total_gap_length: '161368351',
+      spanned_gaps: '661',
+      chromosomes: '25',
+      toplevel_sequences: '640',
+      component_sequences: '36734',
+      gc_percentage: '38.87'
+    },
+    coding_stats: {
+      scientific_name: 'Homo sapiens',
+      coding_genes: '20420',
+      average_genomic_span: '66885.16',
+      average_sequence_length: '3443.15',
+      average_CDS_length: '1139.02',
+      shortest_gene: '8',
+      longest_gene: '2473539',
+      total_transcripts: '153909',
+      coding_transcripts: '100607',
+      transcripts_per_gene: '7.54',
+      coding_transcripts_per_gene: '4.93',
+      total_exons: '1150521',
+      total_coding_exons: '764520',
+      average_exon_length: '243.09',
+      average_coding_exon_length: '149.87',
+      average_exons_per_transcript: '7.48',
+      average_coding_exons_per_coding_transcript: '7.6',
+      total_introns: '996612',
+      average_intron_length: '6367.19'
+    },
+    non_coding_stats: {
+      scientific_name: 'Homo sapiens',
+      non_coding_genes: '23985',
+      small_non_coding_genes: '4870',
+      long_non_coding_genes: '16895',
+      misc_non_coding_genes: '2220',
+      average_genomic_span: '22904.09',
+      average_sequence_length: '1084.17',
+      shortest_gene: '41',
+      longest_gene: '1375317',
+      total_transcripts: '54482',
+      transcripts_per_gene: '2.27',
+      total_exons: '183012',
+      average_exon_length: '347.07',
+      average_exons_per_transcript: '3.36',
+      total_introns: '128530',
+      average_intron_length: '15929.64'
+    },
+    pseudogene_stats: {
+      scientific_name: 'Homo sapiens',
+      pseudogenes: '15217',
+      average_genomic_span: '4124.33',
+      average_sequence_length: '802.62',
+      shortest_gene: '23',
+      longest_gene: '909387',
+      total_transcripts: '18474',
+      transcripts_per_gene: '1.21',
+      total_exons: '43639',
+      average_exon_length: '356.59',
+      average_exons_per_transcript: '2.36',
+      total_introns: '25165',
+      average_intron_length: '4336.85'
+    }
+  },
+  homo_sapiens_GCA_000001405_14: {
+    assembly_stats: {
+      scientific_name: 'Homo sapiens',
+      sex: 'NA',
+      breed_cultivar_strain: 'NA',
```

Comment:
This field can probably just be called "strain". Whichever is the preferred term for this taxonomic group at Ensembl.